oss-cloud-backend/services/databaseService.js

```diff
@@ -22,38 +22,55 @@ module.exports.insertPullRequests = (pullRequests) => {
     pr.status,
     pr.author,
   ]);
-  return mysql.transaction()
-    .query('INSERT INTO contributions (repo, owner, number, link, title, dateCreated, status, author) VALUES ?', [prs])
-    .rollback((e) => { throw e; })
+  return mysql
+    .transaction()
+    .query(
+      'INSERT INTO contributions (repo, owner, number, link, title, dateCreated, status, author) VALUES ?',
+      [prs],
+    )
+    .rollback((e) => {
+      throw e;
+    })
     .commit();
 };
 
-module.exports.getAllContributors = () => mysql.query(
-  'SELECT * FROM contributors',
-);
+module.exports.getContributorsPaging = (params) => mysql
+  .transaction()
+  .query(
+    `SELECT * FROM contributors  ORDER BY ${SqlString.escapeId(

```

Comment:
`sqlstring` library is excessive. Escaping is supported with mysql lib itself -> [link](https://github.com/mysqljs/mysql#escaping-query-values).
Did you consider using it for something else?

Reply:
Mysql lib needs connection object in order to call escape method, which I could not access by using `serverless-mysql` plugin (`mysql.getClient()` method returned `null `everytime). Furthermore, `mysql` lib uses `sqlstring` library in background as can be seen [here](https://github.com/mysqljs/mysql/blob/master/package.json), so that library is already part of our project.

Reply:
Why not `query('SELECT * FROM contributors  ORDER BY ? LIMIT ? ?', args..)`?

Reply:
Changed as requested.